src/main/scala/com/mwz/sonar/scala/util/package.scala

```diff
@@ -0,0 +1,57 @@
+/*
+ * Sonar Scala Plugin
+ * Copyright (C) 2018 All contributors
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+ */
+package com.mwz.sonar.scala
+
+import java.util.Optional
+import scala.language.implicitConversions
+
+/**
+ *  Util methods and conversions
+ *
+ *  @note The Scala.Option <-> Java.Optional conversions were taken from
+ *  https://gist.github.com/julienroubieu/fbb7e1467ab44203a09f
+ */
+package object util {
```

Comment:
There's probably no need to wrap those objects and classes in a package object as suggested by [Alexandru]( https://github.com/alexandru/scala-best-practices/blob/master/sections/2-language-rules.md#218-must-not-include-classes-traits-and-objects-inside-package-objects).

You could just define a chained package instead of the package object like so:
```scala
package com.mwz.sonar.scala
package util

... objects and classes
```

Reply:
Uhm, but then the same should be done with the scoverage package object ?

Reply:
Well, this approach wouldn't work for the scoverage package object because you have a type definition and some vals inside of it.
Perhaps we should just forget about this for now as this is just a minor thing and we can always come back to this and refactor later - definitely not a blocker!

Reply:
Ok, I will leave it as it is for now.